app/src/main/java/com/geeksville/mesh/ui/settings/radio/RadioConfig.kt

```diff
@@ -189,8 +125,35 @@ fun RadioConfigItemList(
         }
     }
 
-    Column(modifier = Modifier.padding(top = 16.dp)) {
-        AdminRoute.entries.forEach { NavButton(it.title, enabled) { onRouteClick(it) } }
+    TitledCard(title = stringResource(R.string.administration), modifier = Modifier.padding(top = 16.dp)) {
+        AdminRoute.entries.forEach { route ->
+            var showDialog by remember { mutableStateOf(false) }
+            if (showDialog) {
+                WarningDialog(
+                    title = "${stringResource(route.title)}?",
+                    onDismiss = { showDialog = false },
+                    onConfirm = { onRouteClick(route) },
+                )
+            }
+
+            val contentColor =
+                when (route) {
+                    AdminRoute.FACTORY_RESET,
+                    AdminRoute.NODEDB_RESET,
+                    -> MaterialTheme.colorScheme.StatusRed
```

Comment:
The contrast on the red from the screenshot seems problematic for users with accessibility issues. I wonder we can do red background with dark or white text?

Reply:
It's pretty problematic even with vision. 

I'd leave them the normal colour, if they aren't 'press me' green, and have a confirm popup on them, I think the standard colour should be fine

Reply:
Reverted in 0fc459ef777f4b2e4c7ab28266afa9a16a8b6928